src/workflows/pyfr/components/steps/Input/index.js

```diff
@@ -256,5 +257,6 @@ export default connect(
   () => ({
     saveSimulation: (simulation) => dispatch(Actions.saveSimulation(simulation)),
     updateSimulation: (simulation) => dispatch(Actions.updateSimulation(simulation)),
+    patchSimulation: (simulation) => dispatch(Actions.patchSimulation(simulation)),
```

Comment:
listing patchSimulation here acts as a self-document for the component. A connect function says "here are the props available from state, here are the actions available from redux"